syntax="proto3";

package iomomi_pb;

message WaterWallEntity {
	float posX = 1;
	float posY = 2;
	float posZ = 3;
	float sizeX = 4;
	float sizeY = 5;
	fixed32 up_plane = 6;
	bool only_initially = 7;
}
